pwndbg/commands/argv.py

```diff
@@ -13,43 +15,34 @@
 import pwndbg.typeinfo
 
 
-@pwndbg.commands.Command
+@pwndbg.commands.ArgparsedCommand(argparse.ArgumentParser(description='Prints out the number of arguments.'))
```

Comment:
Or maybe we should let `ArgparsedCommand` get either `ArgumentParser` instance or `str`.

If `str` would be passed, it would create `parser` on its own passing the string as `description`.
Kinda magic, but less writing for commands without args.

What do you think?